services/app-api/serverless.yml

```diff
@@ -61,10 +62,12 @@ provider:
   name: aws
   runtime: nodejs12.x
   region: us-east-1
+  logs:
+    restApi: true
   iam:
     role:
       path: ${self:custom.iamPath}
-      permissionsBoundary: !Sub arn:aws:iam::${AWS::AccountId}:policy${self:custom.iamPermissionsBoundaryPolicy}
+      permissionsBoundary: ${self:custom.iamPermissionsBoundaryPolicy}
```

Comment:
Example of the change in expected value stored in SSM.  Please read the PR Notes